src/test/java/com/egov/core-services/tests/updatePasswordNoLogin.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Update user
+
```

Comment:
As egov-user has 5 endpoints and 50 tests across all EPs. In this case you can create a folder inside core-services/egov-user/endpointName.feature file like this. This is to just keep standard approach. Inside tests we are referring as individual service and that service feature file should have all endpoints and it's tests.